base_location_geonames_import/__manifest__.py

```diff
@@ -16,9 +16,11 @@
               'Tecnativa,'
               'Odoo Community Association (OCA)',
     'website': 'http://www.akretion.com',
-    'depends': ['base_location'],
+    'depends': ['base', 'base_location'],
```

Comment:
No need to add base (it's implicit)

Reply:
Change also module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers